documentation/assemblies/assembly-uninstalling-openshift.adoc

```diff
@@ -0,0 +1,11 @@
+// This assembly is included in the following assemblies:
```

Comment:
I think this assembly isn't necessary. Instead, have assembly-uninstalling.adoc and include the procedures for openshift and kubernetes in that 1

Reply:
This is the pattern we have followed for the other assemblies, so i'd like to keep it like this now, and then we can restructure so that we create separate books for OpenShift and Kubernetes.